### PR TITLE
Create Staines outline

### PR DESCRIPTION
**Pull request by ghost**
This pull request closes issue #55 
# Adding restaurants and activites
- What activities are you interested in?
- What type of food do you like?

### CODE REVIEW

File: Mawer.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Staines, UK
```
Comment (matteocellucci) [inline Mawer.md:1]: @Mawer Try text title formatting whit # char 👍

File: Mawer.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+Restaurant
```
Comment (matteocellucci) [inline Mawer.md:3]: @Mawer You can create subtitles adding more then one # 🎁

File: Mawer.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+Things to do
+
+- Good size cinema :)
```
Comment (Mawer) [inline Mawer.md]: ##

**Comment by ghost** [comment]
All Done!

**Pull request merged**